portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,28 @@ <h2>My Background</h2>
           </p>
       <h2>My Pets</h2>
       <p>I love my pets and I hope you enjoy their portraits below:</p>
+      <div id="pets">
+          <div class="row">
+            <div class="column">
+                <img src="/images/Fern.jpeg" alt="Fern" style="width:100%">
+                </div>
```

Comment:
You've got mixed tabs and spaces here. It probably looks right in your editor because your tabsize is different from github's default

Reply:
Fixed to 2 spaces per indent in new pull request.